azure/requests.yaml

```diff
@@ -6,7 +6,7 @@ releases:
 - name: ">= 29.5.0"
   requests:
   - name: cilium
-    version: ">= 0.25.2 < 0.26.0"
```

Comment:
this was meant to guarantee that no v29.x would ship 0.26.

Reply:
I am aware of this. We already have a v29.x.x out with Cilium v0.25.2 and there won't be any more v29.x.x releases as we are currently working on v30.0.0.

So technically this is not perfect and you are right, but fixing it properly would make the requests more complex to read.

Reply:
I heavily doubt anyone will bump Cilium to anything behind v0.25.x in a new v29.x.x release, if we'd still make one.

Reply:
> there won't be any more v29.x.x releases

I don't buy that :)

Reply:
Me neither, but let's hope. 😬